### ISSUE TITLE
Two different load balancer protocols 

### ISSUE THREAD

**Issue by DrZoidberg09** [hetznercloud/hcloud-cloud-controller-manager#271]
Hi,

I successfully use the cloud controller to access the load balancer to create an https service with tls termination. Now I have other services that require tcp connection. Is there a good way to access the load balancer to have different services with different protocols? 

I use the server protocal annotation to create the https service. When I create a second load balancer service with the tcp annoation, the https one will be removed and I only have the tcp one.

Any suggestions how and if this is possible to set up would be highly appreciated.

Thanks a lot!

**Comment by fhofherr** [comment]
Hi,

can you please provide a minimal working example showing what you try to achieve?

**Comment by DrZoidberg09** [comment]
What I want to do is running normal web services and a TURN server in the kubernetes cluster. Idea was: https traffic with tls termination in the load balancer and tcp traffic directly to the TURN server.

What I tried at first was creating two different services (in different name spaces) one with https, the other with tcp annotations. Then the second service was the only one working. Meanwhile I have read that the cloud controller only allows one service being connected.
Therefore, I switched to full tcp and tls termination in the kubernetes cluster with cert-manager. That works even better I feel.

However, one question still remains: does the load balancer allow binary tcp traffic to get coturn running? So far I dont have any luck and dont see any traffic at the turn server following the official nginx tutorial.
